hoomd/simulation.py

```diff
@@ -141,6 +141,37 @@ def verbose_run(self, value):
         self._verbose = bool(value)
         self._cpp_sys.enableQuietRun(not self.verbose_run)
 
+    @property
+    def always_compute_pressure(self):
+        """Always compute the virial and pressure.
+
+        By default, HOOMD only computes the virial and pressure on timesteps
+        where it is needed (when :py:class:`hoomd.dump.GSD` writes
```

Comment:
```suggestion
        where it is needed (when :py:class:`hoomd.dump.gsd` writes
```

Reply:
The class is named `hoomd.dump.GSD` per PEP8 naming conventions.